src/components/delegation/DelegationPortal.svelte

```diff
@@ -0,0 +1,90 @@
+<script>
+  import DelegationMetrics from "./DelegationMetrics.svelte";
+  import Delegate from "./Delegate.svelte";
+  import { GET_USER_LAST_DELEGATION } from "../../utils/delegations";
+  import { userAddress } from "../../stores/web3.js";
+  import {
+    veDelegation,
+    delegated,
+    delegationReceived,
+  } from "../../stores/delegation.js";
+  import {
+    getDelegatedVeOcean,
+    getReceivedDelegation,
+  } from "../../utils/delegations.js";
+  import { query } from "svelte-apollo";
+
+  let delegation;
+
+  const init = async () => {
+    let newDelegated = await getDelegatedVeOcean($userAddress);
+    delegated.update(() => newDelegated);
+    let received = await getReceivedDelegation($userAddress);
+    delegationReceived.update(() => received);
+  };
+
+  $: if ($userAddress) {
+    init();
+    delegation = query(GET_USER_LAST_DELEGATION, {
+      variables: { userAddress: $userAddress.toLowerCase() },
+    });
+  }
+
+  $: if ($delegation?.data) {
+    let activeDelegation = $delegation?.data.veDelegations.find(
+      (d) => parseInt(d.updates[0].amount) > 0
+    );
+    let createId = 1;
+    $delegation?.data.veDelegations.forEach((delegation) => {
+      createId += delegation.updates.length;
+    });
+    veDelegation.update(() => {
+      return {
+        ...activeDelegation,
+        createId: createId,
+      };
+    });
+  }
+</script>
+
+<div class={`container`}>
+  <h2 class="title">
+    Allow other wallet to manage your veOCEAN allocation by delegating.
+  </h2>
+  <p class="message">
+    Maximize your active APY and skip transaction fees by delegating to a wallet
+    that efficiently manages your allocation power.
```

Comment:
"Maximize your **Active** APY and skip transaction fees by delegating to a wallet that efficiently manages your allocation power."

Reply:
done